### ISSUE TITLE
TooManyMethods reporting incorrect value

### ISSUE THREAD

**Issue by Rican7** [phpmd/phpmd#275]
On version `2.2.1` (and using `pdepend` version `2.0.6`), running on PHP `5.4.35`, the `TooManyMethods` rule is reporting a value that is lower than the actual count.

As a test, I created two classes. One with `67` (`65` public) methods reported only `33`. And another with `19` (all public) methods reported `12`.

Let me know if there's any other info I can provide. :smiley:

**Comment by ravage84** [comment]
@Rican7 could you please provide us a minified (remove as much as possible, without changing the outcome) of your class?
So we can test this against a concrete example.

**Comment by leewillis77** [comment]
I just hit this. This [bare-bones class](https://gist.github.com/leewillis77/b8b295f4aceb381fe67e) reproduces this error. Despite having 15 methods, phpmd reports:

```
$ phpmd --version
PHPMD 2.1.4
$ phpmd SomeClass.php text codesize,design,cleancode

The class SomeClass has 11 methods. Consider refactoring SomeClass to keep number of methods under 10.
```

**Comment by Rican7** [comment]
Thanks @leewillis77! Very helpful. :smiley:

**Comment by manuelpichler** [comment]
This is quite simple, because getter- and setter-methods are ignored by default, which is common sense in metrics view. But you can change that behavior through the configuration [See here](https://github.com/phpmd/phpmd/blob/master/src/main/resources/rulesets/codesize.xml#L354).

But I must admit that message could be a little bit better.
